daffodil-test/src/test/resources/edu/illinois/ncsa/daffodil/section13/packed/packed.tdml

```diff
@@ -110,5 +153,175 @@
     </tdml:infoset>
 
   </tdml:parserTestCase>  
-      
+
+   <tdml:parserTestCase name="packedCharset01" root="int01" model="s2"
+    description="hex works as a charset encoding">
```

Comment:
These description strings are misleading. We're not treating hex as a charset here. We do have such a charset, but that's not being used here, so these strings are misleading.